src/chain/evm/id.ts

```diff
@@ -38,7 +38,7 @@ export const EvmChainId = {
   SEPOLIA: 11155111,
   // CONSENSUS_ZKEVM_TESTNET: 59140,
   // SCROLL_ALPHA_TESTNET: 534353,
-  BASE_TESTNET: 84532,
```

Comment:
@gaurav-shib Please change the BASE_TESTNET: 84532 (Base Sepolia Testnet)
Whereas BASE_TESTNET: 84531 refers to (Base Goerli Testnet, this is decommissioned)